idaes/core/util/tests/test_model_statistics.py

```diff
@@ -685,6 +689,42 @@ def test_degrees_of_freedom(m):
     assert degrees_of_freedom(m.b2) == -1
 
 
+@pytest.mark.unit
```

Comment:
We should also have test for the functions to collect the number of greybox variables and constraints.

Reply:
added tests